packages/app/src/index.ts

```diff
@@ -5,15 +5,15 @@ export { load, logger } from "@deliverybot/core";
 const pkg = require(path.join(__dirname, "..", "package.json"));
 export const info: { [k: string]: string } = {
   version: pkg.version,
-  homepage: "https://deliverybot.dev",
-  blog: "https://deliverybot.dev/blog/",
-  integrations: "https://deliverybot.dev/integrations/",
-  status: "https://deliverybot.dev/status/",
-  community: "https://spectrum.chat/deliverybot",
-  documentation: "https://deliverybot.dev/docs/",
-  support: "support@deliverybot.dev",
-  github: "https://github.com/deliverybot/deliverybot",
-  terms: "https://deliverybot.dev/terms/terms/",
-  privacy: "https://deliverybot.dev/terms/privacy/",
-  install: "https://github.com/apps/deliverybot/installations/new",
+  homepage: "https://deliverybot.app.medicmobile.org/",
+  blog: "https://medic.org/stories/",
+  integrations: "https://deliverybot.app.medicmobile.org/integrations/",
+  status: "https://deliverybot.app.medicmobile.org/status/",
+  community: "https://forum.communityhealthtoolkit.org/",
+  documentation: "https://docs.communityhealthtoolkit.org/",
+  support: "app-services@medic.org",
+  github: "https://github.com/medic",
+  terms: "https://twitter.com/Medic",
```

Comment:
We could maybe create a medic fork and add a simple licence document for terms. But since this is a fork of a project we did not create, I guess we could leave the delivery bot terms.

Reply:
```suggestion
  terms: "https://deliverybot.dev/terms/terms/",
```